core/src/main/resources/lib/form/advanced/advanced.js

```diff
@@ -22,7 +22,7 @@ Behaviour.specify(
           while (hiddenContent && !hiddenContent.matches("div.advancedBody")) {
             hiddenContent = hiddenContent.nextElementSibling;
           }
-          tr = parentContainer.closest(".tr");
+          tr = parentContainer.closest("[data-type='advanced-button-group']");
```

Comment:
does there need to be guidance in the design library on this approach?

Reply:
Yeah, I'll write something up for the Conventions page